css/properties/-moz-border-bottom-colors.json

```diff
@@ -20,16 +20,30 @@
             "edge_mobile": {
               "version_added": false
             },
-            "firefox": {
-              "version_added": "1",
-              "version_removed": "59",
-              "notes": "From Firefox 59, the property is limited to usage in chrome code only."
-            },
-            "firefox_android": {
-              "version_added": "4",
-              "version_removed": "59",
-              "notes": "From Firefox 59, the property is limited to usage in chrome code only."
-            },
+            "firefox": [
+              {
+                "version_added": "60",
```

Comment:
This should probably be `59` instead of `60` since it has been available to internal code in Firefox 59.0.